radis/lbl/factory.py

```diff
@@ -529,38 +530,29 @@ def __init__(
             if optimization is None:
                 if self.verbose >= 3:
                     printg(
-                        "LDM algorithm not used. Defaulting truncation from {0} to 10".format(
+                        "LDM algorithm not used. Defaulting truncation from {0} to 0".format(
```

Comment:
I made an error here. Default truncation should be 300 cm-1. And default neighbor likes should be 0. Algorithm shouldn't matter (DIT /Voigt and LBL/Voigt have same truncation,  only DIT/FFT should raise an error if truncation as its not implemented yet)

Reply:
DIT/Voigt with 300 cm-1 truncation will be feasible, but I'm not sure about LBL/Voigt. As seen from the benchmarks before, LBL is independent of spectral range i.e. neighbour_lines, but is dependent on the truncation. And for a truncation=300cm-1, the calculation time and memory will be expensive. So a default 300 cm-1 may not be feasible.

Reply:
Good point for the performance drop. But at least the physics remain the same, so it's a good reason to have 300 cm-1 always  (similar to the remark of @dcmvdbekerom  in #340  : we don't want to change the physical output without the user knowing / deciding) 

Given DIT/Voigt is the default, it should be fine in most cases. 

And anyway, if the predict_time() is given before the broadening step, the user will know. 
We can add an extra PerformanceWarning if predicted calculation time exceeds a few minutes.